lib/Events/ModifyParticipantEvent.php

```diff
@@ -48,14 +48,14 @@ public function getParameter(): string {
 	/**
 	 * @return int|string|bool
 	 */
-	public function getNewValue() {
+	public function getNewValue(): bool|int|string {
```

Comment:
Unless the type is added to the constructor, I don't think we should type the getter.

Reply:
It's addressed in the next commit